[datadog_synthetics] Add TOTP Parameters for Global Variables (#1708)
DataDog/terraform-provider-datadog: docs/resources/synthetics_global_variable.md

### Optional

- `description` (String) Description of the global variable.

> - `options` (Block List, Max: 1) Additional options to add for the variable, such as an MFA token. (see [below for nested schema](#nestedblock--options))

Reviewer: ```suggestion
- `options` (Block List, Max: 1) Additional options for the variable, such as a MFA token. (see [below for nested schema](#nestedblock--options))
```
